dns/zone-configs/k8s.io._0_base.yaml

```diff
@@ -110,7 +110,7 @@ releases:
 artifacts:
   type: A
   values:
-  - 35.244.192.77
+  - 34.110.216.12
```

Comment:
Do we need to define a short TTL to ensure DNS update is quickly propagated ?

Reply:
That's a good idea, if we can.  I put this through in case you didn't have DNS permissions, but it looks like you do.

On IPv6, I wasn't sure whether we wanted to change "two things at once", in case IPv6 broke some people... happy to do it at the same time though.

Reply:
For the TTL, you can add a `ttl` field according to https://github.com/octodns/octodns/blob/v0.9.9/docs/dynamic_records.md.

For IPv6 we can do it later.